R/get_source_expressions.R

```diff
@@ -290,6 +290,7 @@ fix_eq_assigns <- function(pc) {
 
   eq_assign_locs <- which(pc$token == "EQ_ASSIGN")
   if (length(eq_assign_locs) == 0L ||
+    length(pc$token) %in% eq_assign_locs ||
```

Comment:
Could you use `nrow(pc) %in% eq_assign_locs`, or add a comment to state that this is checking whether the equal-assignment is the final entry in the parsed content

Reply:
Sure